product/roundhouse.tasks/Roundhouse.cs

```diff
@@ -142,12 +153,14 @@ public string RecoveryMode
             set
             {
                 RecoveryMode result;
-                if (!Enum.TryParse(value, true, out result))
+                if (Enum.TryParse(value, true, out result))
                 {
-                    throw new ArgumentOutOfRangeException("value", value, "The value of 'RecoveryMode' must be one of these values: 'NoChange', 'Simple' or 'Full'.");
+                    this.recoveryMode = result;
+                }
+                else
+                {
+                    this.loggingHelper.LogError("The value of 'RecoveryMode' must be one of these values: 'NoChange', 'Simple' or 'Full'. Actual value was '{0}'.", value);

```

Comment:
Well in the case of this code updates, if ContinueOnError is set, it will be set to `NoChange` (the default) - https://github.com/chucknorris/roundhouse/blob/master/product/roundhouse/databases/RecoveryMode.cs#L5.